hax/hax/util.py

```diff
@@ -1988,14 +1996,11 @@ def get_process_based_node_state(self, node_fid: Fid) -> str:
             p_fid = match_result.group(1)
             # Counting total number of configured processes
             total_processes += 1
-
-            p_key = f"processes/{p_fid}"
-            process = self.kv.kv_get(p_key, recurse=False)
-            if not process:
+            # if process is not found then it is considered as offline
+            if p_fid not in all_procs:
                 continue
-            state = json.loads(process['Value'])['state']
             # Checking if status is M0_CONF_HA_PROCESS_STARTED
-            if state == ha_process_events[1]:
+            if all_procs[p_fid] == ha_process_events[1]:
```

Comment:
Why not use `m0HaProcessEvent.M0_CONF_HA_PROCESS_STARTED` instead of `ha_process_events[1]`, will be more readable.

Reply:
done